src/client/src/orangehrmCoreOAuthPlugin/pages/registerOAuthClient/SaveOAuthClient.vue

```diff
@@ -22,12 +22,16 @@
   <div class="orangehrm-background-container">
     <div class="orangehrm-card-container">
       <oxd-text tag="h6" class="orangehrm-main-title">
-        {{ $t('admin.add_oauth_client') }}
+        {{
+          editMode
+            ? $t('admin.edit_oauth_client')
+            : $t('admin.add_oauth_client')
+        }}
       </oxd-text>
 
       <oxd-divider />
 
-      <oxd-form :loading="isLoading" @submit-valid="onSave">
+      <oxd-form novalidate="true" :loading="isLoading" @submit-valid="onSave">
```

Comment:
@Super-Chama when do we need `novalidate="true"`, it came from OAuth edit screen

Reply:
no need it's already added in oxd form https://github.com/orangehrm/oxd/blob/main/components/src/core/components/Form/Form.vue#L25